rust/libnewsboat/src/fmtstrformatter/mod.rs

```diff
@@ -131,7 +131,7 @@ impl FmtStrFormatter {
         result: &mut LimitedString,
     ) {
         match self.fmts.get(&cond) {
-            Some(value) if !value.is_empty() => {
+            Some(value) if !value.trim().is_empty() => {
```

Comment:
This has to be tested and documented:
1) please add a test (the same one for Rust and C++);
2) please mention this change in the CHANGELOG;
3) add a note about this somewhere in the docs.

Reply:
Done, done, and done.